site/includes/datemodified.hbs

```diff
@@ -1,4 +1,11 @@
 <div class="row pagedetails">
+<div class="datemod col-xs-12 mrgn-tp-lg">
+		<dl id="wb-dtmd">
+			<dt>{{{i18n "tmpl-date-mod"}}}{{{i18n "space"}}}</dt>
+			<dd><time>{{page.dateModified}}</time></dd>
+		</dl>
+	</div>
+	<div class="clearfix"></div>
```

Comment:
Bootlint error: A row can only have column

https://github.com/twbs/bootlint/wiki/E013

A solution will be to put the following content in a columns

Reply:
@duboisp I believe the changes I made in #1290 addressed that error.